env_api/core/models/tiramisu_program.py

```diff
@@ -40,31 +43,35 @@ def load_code_lines(self):
             # the lines of the real function to execute legality code
             func_name = self.name
             file_name = func_name + "_generator.cpp"
-            file_path = (Config.config.dataset.benchmark_cpp_files
-                         if Config.config.dataset.is_benchmark else Config.
-                         config.dataset.path) + func_name + "/" + file_name
+            if (cfg.Config.config is None):
```

Comment:
Config.config will never be none , because it is an obligation to initialize it before creating the tiramisuapi object , i think this condition is always true .